src/lib/djerba/plugins/wgts/snv_indel/tools.py

```diff
@@ -106,10 +106,16 @@ def annotate_maf(self, maf_path):
         return factory.get_annotator(self.work_dir, self.config).annotate_maf(maf_path)
 
     def convert_vaf_plot(self):
-        """Read VAF plot from file and return as a base64 string"""
+        """
+        Read VAF plot from file if it exists and return as a base64 string
+        Else, return False
+        """
         image_converter = converter(self.log_level, self.log_path)
         plot_path = os.path.join(self.work_dir, sic.VAF_PLOT_FILENAME)
-        vaf_plot = image_converter.convert_svg(plot_path, 'CNV plot')
+        if self.workspace.has_file(sic.VAF_PLOT_FILENAME):
+            vaf_plot = image_converter.convert_svg(plot_path, 'CNV plot')
+        else:
+            vaf_plot = False
```

Comment:
`None` is the standard Python way to express this -- it evaluates to False in boolean comparisons, but doesn't suggest you're dealing with a variable which is either True or False